book/src/developers/quick_setup.md

```diff
@@ -59,7 +61,95 @@ View CLI options:
 cargo run -- --help
 ```
 
-### Run locally
+## Building on Arch Based Systems
+
+Before starting, update your system.
+
+```bash
+sudo pacman -Syu
+```
+
+Then, install rust.
+
+```bash
+sudo pacman -S rustup
+rustup install stable
+```
+
+To check that the rust toolchain was successfully installed, run:
+
+```bash
+rustc --version
+```
+
+You should see something like:
+
+```bash
+rustc 1.71.1 (eb26296b5 2023-08-03)
+```
+
+Next, install the required dependencies:
+
+```bash
+sudo pacman -S openssl clang pkg-config base-devel llvm git
+```
+
+You might want to make sure that `clang` and `llvm` version is lower than 16.
+
+```bash
+$ clang --version
+    clang version 15.0.7
+    Target: x86_64-pc-linux-gnu
+    Thread model: posix
+    InstalledDir: /usr/sbin
+    
+$ llvm-config --version
+    15.0.7
+```
+
+gcc 13 generates some problems when building a rust library called `rocksdb-sys`. For that reason, you need to downgrade it to version 12. Run this command:
```

Comment:
Could you try to bump `rocksdb` version in Trin to [v0.21.0](https://github.com/rust-rocksdb/rust-rocksdb/releases/tag/v0.21.0) and check if you get the same issue with gcc?

Reply:
It works!

Just oppened #839.

Reply:
Cool, I guess we now want to remove the `gcc` comments from the guide.